src/react-components/debug-panel/RtcDebugPanel.js

```diff
@@ -0,0 +1,699 @@
+    kind: data.kind
+  };
+  return (
+    <DataPanel title={`${data?.kind} ${title}`} data={props}>
+      {data?.track && <DataPanel key={data.id} data={data.track} title={`track`} />}
+      {stats && (
+        <DataPanel
+          key={stats.rtpStats?.id}
+          graphData={stats.graph.slice()}
+          width={"120px"}
+          height={"50px"}
+          color={"green"}
+          speed={stats.speed}
+          data={stats.rtpStats}
+          xAxis={xAxis}
+          yAxis={yAxis}
+          title={`stats`}
+        />
+      )}
+    </DataPanel>
+  );
+}
+
+TrackStatsPanel.propTypes = {
+  title: PropTypes.string,
+  data: PropTypes.object,
+  xAxis: PropTypes.string,
+  yAxis: PropTypes.string,
+  stats: PropTypes.object
+};
+
+function TransportPanel({ title, data, candidates, producers, consumers, isButtonEnabled, onRestart }) {
+  return (
+    <CollapsiblePanel key={title} title={title} border url={`${MEDIASOUP_DOC_BASE_URL}#Transport`} data={data}>
+      <PanelMessageButton
+        id={"rtcDebugPanel.restartIceButton"}
+        onClick={onRestart}
+        disabled={!isButtonEnabled}
+        primary
+      />
+      {candidates?.length > 0 && (
+        <CollapsiblePanel key={`Candidates`} title={`Candidates`} row url={`${MDN_DOC_BASE_URL}RTCIceCandidate`} border>
+          {candidates}
+        </CollapsiblePanel>
+      )}
+      {producers?.length > 0 && (
+        <CollapsiblePanel
+          key={`Local producers`}
+          title={`Local producers`}
+          row
+          url={`${MEDIASOUP_DOC_BASE_URL}#Producer`}
+          border
+        >
+          {producers}
+        </CollapsiblePanel>
+      )}
+      {consumers}
+    </CollapsiblePanel>
+  );
+}
+
+TransportPanel.propTypes = {
+  candidates: PropTypes.array,
+  producers: PropTypes.array,
+  consumers: PropTypes.array,
+  data: PropTypes.object,
+  onRestart: PropTypes.func,
+  title: PropTypes.string,
+  isButtonEnabled: PropTypes.bool
+};
+
+function SignalingPanel({ data, onConnect, onDisconnect }) {
+  return (
+    <CollapsiblePanel title={"Signaling"} border data={data}>
+      <PanelMessageButton
+        id={data.connected ? "rtcDebugPanel.disconnectSignalingButton" : "rtcDebugPanel.connectSignalingButton"}
+        onClick={() => {
+          if (data.connected) {
+            onDisconnect();
+          } else {
+            onConnect();
+          }
+        }}
+        disabled={false}
+        primary
+      />
+    </CollapsiblePanel>
+  );
+}
+
+SignalingPanel.propTypes = {
+  data: PropTypes.object,
+  onConnect: PropTypes.func,
+  onDisconnect: PropTypes.func
+};
+
+export default class RtcDebugPanel extends Component {
+  static propTypes = {
+    avatarId: PropTypes.string,
+    store: PropTypes.object,
+    scene: PropTypes.object,
+    intl: PropTypes.object,
+    presences: PropTypes.object,
+    history: PropTypes.object,
+    sessionId: PropTypes.string
+  };
+
+  constructor() {
+    super();
+
+    this.state = {};
```

Comment:
In react, you want to specify default values for this.state so it has defaults to update. 

This line would say:
```javascript
this.state = {
    showPanel: false, // default option
    log: [],
    // other values
}
```